signup/signup.php

```diff
@@ -81,12 +71,12 @@
 			<h3 class="form-subheading">Background Information</h3>
 
 			<ul class="dcf-pl-0">
-				<!-- TODO: NOTE THAT THIS WAS COMMENTED OUT, WE CURRENTLY DONT LET THEM SAY THE LANGUAGE THEY REQUIRED, WE LET IT DEFAULT TO ENGLISH
+				<!-- TODO: NOTE THAT THIS WAS COMMENTED OUT, WE CURRENTLY DONT LET THEM SAY THE LANGUAGE THEY REQUIRED, WE LET IT DEFAULT TO ENGLISH -->
 				<li class="form-radio" id="language">
 					<label for="language" class="form-required">Which language will you require?</label>
 					<div>
 						<div class="dcf-btn-group" data-toggle="buttons">
-							NOTE: the values here are the ISO 639-2/T specfication for language codes (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)
+							<!-- NOTE: the values here are the ISO 639-2/T specfication for language codes (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) -->
 							<label class="dcf-btn dcf-btn-secondary" 
```

Comment:
Are these 4 languages still the 4 they want displayed? Also, does this selection default to English?

Reply:
It defaults to English now.

Reply:
Tracy just got back to me and let me know these are good options for now.